bspump/elasticsearch/connection.py

```diff
@@ -47,11 +47,8 @@ def __init__(self, connection, index, max_size):
 		self.FailLogMaxSize = connection.FailLogMaxSize
 		self.FilterPath = connection.FilterPath
 
-		# Get credentials
-		username = connection.Config.get('username')
-		password = connection.Config.get('password')
-		api_key = connection.Config.get('api_key')
-		url = connection.Config.get('url')
+		# Get credentials for authorization
+		username, password, api_key, url = get_credentials_from_config(self.Config)
```

Comment:
`ElasticSearchBulk` does not have attribute `Config`. Use `connection.Config`.

Reply:
ok, thanks